config/routes.rb

```diff
@@ -1,12 +1,17 @@
 Rails.application.routes.draw do
 
-root 'tasks#index'
+  root 'tasks#index'
+
+  get '/tasks/:id/edit' => 'tasks#edit'
+  patch '/tasks/:id' => 'tasks#update'
+  get '/tasks'=> 'tasks#index'
+  get '/tasks/new' => 'tasks#new'
+  delete '/tasks' => 'tasks#destroy'
```

Comment:
It might be a good idea to have this route use a URL path that includes the ID parameter, like: `delete '/tasks/:id' => 'tasks#destroy'`.
If the delete buttons in your `index.html.erb` file are working, then it's all good. But usually for delete routes we have the ID included in the URL.